clang/test/CodeGenHLSL/enable-16bit-types.hlsl

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -fnative-half-type -finclude-default-header -triple dxil-pc-shadermodel6.3-library -emit-llvm -disable-llvm-passes -o - %s | FileCheck %s --check-prefix=FLAG
```

Comment:
NIT: For future cleanup  and test organization could we move the Shader Flag tests into their own directory like the Builtins have their own directory? Not asking for this PR.